app/controllers/application.js

```diff
@@ -7,7 +7,7 @@ export default Controller.extend({
   dj             : service(),
   hifi           : service(),
   currentStream  : service(),
-  links: [ { 'href': null, 'nav-slug': 'listen', 'title': 'Listen'}, { 'href': null, 'nav-slug': 'playlist-history', 'title': 'Playlist History'} ],
+  links: [ { 'href': null, 'nav-slug': 'listen', 'title': 'LISTEN'}, { 'href': null, 'nav-slug': 'playlist-history', 'title': 'PLAY HISTORY'} ],
```

Comment:
I think we should be using CSS to transform text into caps using `text-transform: uppercase` rather than requiring the actual text be all uppercase. Since the fact that it's capital is just a visual treatment/style I think it's good to keep all that in the CSS and not dependent on the contents. I've seen this elsewhere too 

cc: @msbeethoven

Reply:
done